dashboard/app/views/admin_search/search_for_teachers.html.haml

```diff
@@ -64,3 +63,10 @@
             %span= field
 - else
   No teachers met the specified search criteria.
+
+
+:javascript
+  var email = '#{ params[:emailFilter]}';
+  var address = '#{ params[:addressFilter]}';
+  document.getElementById("emailFilter").defaultValue = email
```

Comment:
Is there a reason you're doing this here rather than doing `%input{name: "emailFilter", id: "emailFilter", value: params[:emailFilter] }` above?